src/partials/products.html

```diff
@@ -0,0 +1,62 @@
+<section class="products container">
+    <div class="section">
```

Comment:
Техничесский класс `container` не должен быть прописанным симантичесскому тегу `section`, так как урежет размер всей секции.
Для примера, `background` поставленный даже на класс `products`, будет урезан по размеру контейнера, так как он прописан сразу же для секции, а не отдельным блоком.
И мы удалили дополнительный класс `section`, поскольку у всех секций слишком разные пораметры, чтобы их прописовать в один класс. Как раз на этом место и должен стоять `container`.

Как минимум классы должны быть распределены таким образом, если же ты хотел использовать класс `section` в своих целях:
```suggestion
<section class="products section">
    <div class="container">
```

Если что-то не понятно, то пиши - поможем объясним =)